tests/test_visitors/test_ast/test_loops/test_loops/test_implicit_yield_from.py

```diff
@@ -31,17 +31,43 @@ def test_implicit_yield_from(
     code,
     template,
     default_options,
-    mode,
+    regular_wrapper,
 ):
     """Ensures that implicit ``yield from`` are not allowed."""
-    tree = parse_ast_tree(mode(template.format(code)))
+    tree = parse_ast_tree(regular_wrapper(template.format(code)))
 
     visitor = WrongLoopDefinitionVisitor(default_options, tree=tree)
     visitor.run()
 
     assert_errors(visitor, [ImplicitYieldFromViolation])
 
 
+@pytest.mark.parametrize('template', [
+    for_loop_template,
+])
+@pytest.mark.parametrize('code', [
+    'yield',
+    'yield None',
+    'yield index',
+    'yield 10',
+])
+def test_async_implicit_yield_from(
+    assert_errors,
+    parse_ast_tree,
+    code,
+    template,
+    default_options,
+    async_wrapper,
```

Comment:
`mode` maybe?

Reply:
No. `mode` add both `sync` and `async`.  In `test_implicit_yield_from` test I removed `mode` and used `regular_wrapper`. And here only `async_wrapper`